meinberlin/assets/scss/components_user_facing/_live_questions.scss

```diff
@@ -7,3 +7,29 @@
     margin-left: auto;
     margin-bottom: .625em;
 }
+
+.live_question__progress {
+    display: flex;
+    gap: 8px;
+    margin-bottom: 8px;
+    position: relative;
+}
+
+.live_question__progress-bar {
+    background-color: $gray-lighter;
+    border-radius: 10px;
+    height: 100%;
+    position: absolute;
+    z-index: 0;
+}
+
+.live_question__progress-text {
+    padding-left: 8px;
+    padding-right: 8px; 
```

Comment:
can you use `em` or `rem` more, where it makes sense please? I think we prefer that over px, unless its some specific number or similar.